tests/unit/test_schemas.py

```diff
@@ -27,13 +27,30 @@ def stories_schema():
     return YAML(typ="safe").load(open(schema_file, encoding="utf-8"))
 
 
+@pytest.fixture
```

Comment:
Would it make sense to set a scope to this fixture?

Reply:
Probably not. Fixture scope takes care of clearing fixture when pytest leaves the given scope (module, package, etc.), and this particular fixture is not harmless, does not spoil other tests or fixtures, and it's not in conflict with any other fixture.

Reply:
thanks Milos.